vta/python/vta/testing/util.py

```diff
@@ -31,7 +31,7 @@ def run(run_func):
     """
     env = get_env()
 
-    if env.TARGET == "sim":
+    if env.TARGET == "sim" or env.TARGET == "tsim":
```

Comment:
`if env.TARGET in ["sim", "tsim"]`

Reply:
Fixed [here](https://github.com/vegaluisjose/tvm/commit/b3a760fb287bd2ed9a2fdd12e306200e7698914b)